.buildkite/scripts/steps/cloud/purge_deployments.ts

```diff
@@ -45,6 +45,7 @@ for (const deployment of prDeployments) {
   } catch (ex) {
     console.error(ex.toString());
     // deploymentsToPurge.push(deployment); // TODO should we delete on error?
+    process.exit(1);
```

Comment:
You should probably keep track of whether or not an error occurred, and then exit at the very end of the script, right? Otherwise, an error on the first deployment would keep all of the other ones from getting purged?

Reply:
Pushed https://github.com/elastic/kibana/pull/137733/commits/6b801d542361e6fe95f51f56f059a24eedae37a4